utest/run.py

```diff
@@ -93,10 +82,24 @@ def usage_exit(msg=None):
 
 
 if __name__ == '__main__':
-    docs, vrbst = parse_args(sys.argv[1:])
-    tests = get_tests()
+    parser = argparse.ArgumentParser(add_help=False, description=__doc__)
+    parser.add_argument('-I', '--interpreter', default=sys.executable)
+    parser.add_argument('-h', '--help', action='store_true')
+    parser.add_argument('-q', dest='vrbst', action='store_const', const=0)
+    parser.add_argument('-v', dest='vrbst', action='store_const', const=2)
+    parser.add_argument('-d', dest='docs', action='store_true')
+    parser.add_argument(dest='directory', nargs='?', action='store', default=None)
+    parser.set_defaults(vrbst=1)
```

Comment:
:+1: for `argparse`. These changes are unrelated to the actual fix and would be better to have them in a separate PR, but no need to separate them now.